src/v2/Apps/lang/en-us.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
should we merge this file with the other one outside `Apps/`?

Reply:
I suggested above that we output to `v2/__i18n__`, like we do for relay generated fragments (`v2/__generated__`)

Reply:
I think this should live in `v2/System/i18n/lang`

Reply:
I moved this to `v2/System/i18n/lang` for now, but I can change this in case we decide we really want to have multiple locale files